app.py

```diff
@@ -0,0 +1,73 @@
+import spacy
+from spacy.lang.de.examples import sentences 
+import pickle
+import os.path
+import requests
+
+def load_pickle_files():
+	with open('files/masculine_suffix.pkl', 'rb') as handle:
+	    suffixes_masc = pickle.load(handle)
+
+	with open('files/feminine_suffix.pkl', 'rb') as handle:
+	    suffixes_fem = pickle.load(handle)
+
+	with open('files/neutral_suffix.pkl', 'rb') as handle:
+	    suffixes_neut = pickle.load(handle)
+
+	return suffixes_masc, suffixes_fem, suffixes_neut
+
+def calculate_gender(noun, suffixes_masc, suffixes_fem, suffixes_neut):
```

Comment:
Use typehints (5.2 https://ufal.mff.cuni.cz/courses/npfl096-2014-su/project).

Here it should be probably 
```suggestion
def calculate_gender(noun: str, suffixes_masc: List[str], suffixes_fem: List[str], suffixes_neut: List[str]) -> str:
```

Reply:
fixed